bitsandbytes/cuda_setup/main.py

```diff
@@ -112,9 +113,13 @@ def manual_override(self):
         if not override_value:
             return
 
-        binary_name = self.binary_name.rsplit(".", 1)[0]
-        # TODO: what's the magic value `-3` here?
-        self.binary_name = binary_name[:-3] + f'{override_value}{DYNAMIC_LIBRARY_SUFFIX}'
+        binary_name_stem, _, binary_name_ext = self.binary_name.rpartition(".")
+        # `binary_name_stem` will now be e.g. `/foo/bar/libbitsandbytes_cuda118`;
+        # let's remove any trailing numbers:
+        binary_name_stem = re.sub(r"\d+$", "", binary_name_stem)
```

Comment:
how about to use `rstrip()`?
`binary_name_stem = binary_name_stem.rstrip("0123456789")"`

Reply:
Hey, great idea! 💪